data/categories.txt

```diff
@@ -16515,3 +16515,44 @@ uk:Інвалідний візок|5Інвалід
 vi:Xe lăn
 zh-Hans:轮椅
 zh-Hant:輪椅
+
+amenity-social_facility
+en:Social Facility
+ar:المرفق الاجتماعي
+be:Сацыяльная ўстанова
+bg:Социално съоръжение
+ca:Equipament Social
+cs:Sociální zařízení
+da:Social facilitet
+de:Soziale Einrichtung
+el:Κοινωνική διευκόλυνση
+es:Servicio social
+et:Sotsiaalne rajatis
+eu:Gizarte-ekipamendua
+fa:تسهیلات اجتماعی
+fi:Sosiaalinen väline
+fr:Établissement de services sociaux
+he:מתקן חברתי
```

Comment:
The translation on iD is:
```suggestion
he:מתקן שירותי רווחה
```

I'm really not sure of this translation but can't think of anything better. 'Facility' is hard to translate to Hebrew.